external/source/exploits/drunkpotato/RogueWinRM dll/ReflectiveDLLInjection.h

```diff
@@ -0,0 +1,53 @@
+//===============================================================================================//

```

Comment:
RogueWinRM should be updated to leverage the existing ReflectiveDLLInjection code that is already included within Metasploit under `external/source/ReflectiveDLLInjection` (which is a submodule you'll need to initialize FYI).

Reply:
Fixed by commit [37dffaf7038c09168716210397c2985f85cea0e0](https://github.com/C4ssandre/metasploit-framework/commit/37dffaf7038c09168716210397c2985f85cea0e0)